core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/hybrid/HybridJDBCConstants.java

```diff
@@ -161,4 +161,13 @@ public class HybridJDBCConstants {
 
     public static final String COUNT_INTERNAL_ONLY_ROLES_SQL = "SELECT COUNT(UM_ID) AS RESULT FROM UM_HYBRID_ROLE " +
             "WHERE UM_ROLE_NAME NOT LIKE 'Application%' AND UM_ROLE_NAME LIKE ? AND UM_TENANT_ID = ?";
+
+    public static final String GET_GROUP_ID = "SELECT UM_ID FROM UM_HYBRID_GROUP_ROLE WHERE UM_GROUP_NAME =? " +
+            "AND UM_TENANT_ID=?";
```

Comment:
```suggestion
    public static final String GET_GROUP_ID = "SELECT UM_ID FROM UM_HYBRID_GROUP_ROLE WHERE UM_GROUP_NAME = ? " +
            "AND UM_TENANT_ID = ?";
```
Shall we format other 2 DB queries as well?

Reply:
Fixed with ee2c01d7a4bbe760b4090f5652a8ae87cdbc987e